cogip/tools/detector/__main__.py

```diff
@@ -24,6 +24,13 @@ def main_opt(
         help="Server URL",
         envvar="COGIP_SERVER_URL"
     ),
+    id: int = typer.Option(
+        1,
+        "-i", "--id",
+        min=1,
+        help="Robot ID.",
+        envvar=["ROBOT_ID", "DETECTOR_ID"]
```

Comment:
No.
Environment variables used to set options are all prefixed by the tool name (DETECTOR_*, PLANNER_*, etc.).
So it is the same for id parameters, and because copilot, detector and robotcam have the same id, I added a common variable ROBOT_ID in all tools running on the robot.